### PR TITLE
AST-1946 - accessibility css changes for coupon code label

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description

Found accessibility issue on quick view link and coupon field label issue.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by nileshc-bsf — APPROVED** [review]

File: sass/site/compatibility/woocommerce/woocommerce-layout-grid.scss [hunk 1]
```diff
@@ -353,6 +353,14 @@
...
 		position: relative;
 	}
+
+	#content {
+		.coupon {
+			.ast-coupon-label {
+				display: none;
+			}
+		}
+	}
```
Comment (imnavanath) [inline sass/site/compatibility/woocommerce/woocommerce-layout-grid.scss:363]:
```suggestion
	.coupon {
		.ast-coupon-label {
			display: none;
		}
	}
```

Comment (imnavanath) [reply sass/site/compatibility/woocommerce/woocommerce-layout-grid.scss:363]: No need of extra dependency as this is unique selector.

Comment (imnavanath) [reply sass/site/compatibility/woocommerce/woocommerce-layout-grid.scss:363]: If we adding the markup from Addon then why we are adding this CSS from theme?? https://github.com/brainstormforce/astra-addon/pull/1350/files#diff-c168383fb11e852266a670adb9e1ae7d78c4d7449b97e4b447768ac65320de9cR3301

Comment (GauravKhupse) [reply sass/site/compatibility/woocommerce/woocommerce-layout-grid.scss:363]: Yes!, I will close this PR, and I will add CSS from the addon.

**Comment by GauravKhupse** [comment]
I will add css from astra-addon so I will close this PR

**Pull request closed without merging**